backup/Dockerfile

```diff
@@ -36,4 +39,9 @@ ENV SCHEDULE **None**
 ADD run.sh run.sh
 ADD backup.sh backup.sh
 
+ENV USER 88735
```

Comment:
why this user?

Reply:
It's just an arbitrary numeric user. Got any other preference?

Reply:
Maybe we should reuse 65532, which is used by our other images? I have no strong preference.

Reply:
Sure. I was kind of intentionally choosing something random and different from the others, but I'm fine with the same ID as well.